taar_api/hbase/management/commands/populate_sample_data.py

```diff
@@ -0,0 +1,129 @@
+import json
+from django.core.management.base import BaseCommand
+from django.conf import settings
+from happybase.connection import Connection
+
+
+class Command(BaseCommand):
+    help = 'Populate the HBase table with some sample data'
+
+    def handle(self, *args, **options):
+        conn = Connection(settings.HBASE_HOST)
+        if settings.HBASE_TABLE.encode() not in conn.tables():
+            self.stderr.write(
+                self.style.ERROR('Table "%s" not found.' % settings.HBASE_TABLE))
+        table = conn.table(settings.HBASE_TABLE)
+        data = json.dumps({
```

Comment:
Each sample in the real table contains much less data, see [here](https://github.com/mozilla/telemetry-batch-view/blob/master/src/test/scala/com/mozilla/telemetry/views/HBaseAddonRecommenderViewTest.scala#L10-L22). Do we care?